src/api/app/models/owner_search/container.rb

```diff
@@ -0,0 +1,46 @@
+module OwnerSearch
+  class Container < Base
+    def for(container)
+      if container.is_a?(Project)
+        project = container
```

Comment:
maybe:

```
project = container.is_a?(Project) ? container : container.project
```

Reply:
maybe - but maybe not. I don't like these ? : - statements

Reply:
what's the difference between if and "elvis" (?:) operator next to typing 9 more characters?

Reply:
I thought I already said so: I don't like elvis (my son was born on the night of elvis' 30 death  anniversary and it was a very traumatic night :)